core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.controller.ts

```diff
@@ -102,13 +86,53 @@ angular.module('oppia').controller('QuestionSuggestionReviewModalController', [
       });
     };
 
+    $scope.reviewable = reviewable;
+    $scope.misconceptionsBySkill = misconceptionsBySkill;
+    $scope.currentSuggestionId = suggestionId;
+
+    $scope.currentSuggestion = suggestionIdToContribution[suggestionId];
+    delete suggestionIdToContribution[suggestionId];
```

Comment:
Why is this needed?

Reply:
We delete the currentSuggestion from the `suggestionIdtoContribution` as it contains all contributions but we only need the 'other' contributions for the remainingContributionIds array.